conference/talk_voting.py

```diff
@@ -107,7 +107,7 @@ def find_talks(user, conference, extra_filters):
         .annotate(
             can_vote=Case(
                 When(created_by=user, then=Value(False)),
-                When(speakers__user__in=[user], then=Value(False)),
+                When(speakers__user__username__icontains=[user.username], then=Value(False)),
```

Comment:
why icontains?

Reply:
Well, that actually fixes the query - before it would return a separate row for each speaker of the talk - so if you're a co-speaker, you'd get the talk twice, once with can_vote set to `True` and once to `False`.

Reply:
I'm still puzzled about this, but let's leave it as is. :)